docs/testing-vale.rst

```diff
@@ -0,0 +1,21 @@
+.. _testing_vale:
+
+Does it warn about sql? => no
+
+Does it warn about Postgres? => yes
+
+.. NOTE::
+
+   Does it warn about Postgres in see also? => yes
+
+.. SEEALSO::
+
+   Does it warn about Postgres in see also? => no
```

Comment:
I noticed that sometimes it only reports on the first occurence and not on subsequent ones. Not sure why that's the case.

Reply:
I tried some additional things but it's not fully conclusive to me...

* it seems to consistently ignore `NOTE`
* it seems to consistently work in `SEEALSO`
* list behaviour is inclusive... sometimes it reports multiple times, sometimes it does on last occurrence, did not see first occurrence yet 🤷 

Will drop this file for now to make checks pass and so that this PR can be moved to ready for review